layers/meta-resin-raspberrypi/conf/samples/bblayers.conf.sample

```diff
@@ -14,7 +14,7 @@ BBLAYERS ?= " \
     ${TOPDIR}/../layers/meta-openembedded/meta-python \
     ${TOPDIR}/../layers/meta-raspberrypi \
     ${TOPDIR}/../layers/meta-resin/meta-resin-common \
-    ${TOPDIR}/../layers/meta-resin/meta-resin-jethro \
```

Comment:
This needs some more changes. See: https://github.com/resin-os/resin-raspberrypi/commit/d40fc952259a198ad4ec20ce423f0e074e9c19cf . Otherwise the build system will migrate the conf at every new build.